.env.test

```diff
@@ -0,0 +1 @@
+export EMAIL_DEFAULT_FROM='no-reply@nourish.example.com'
```

Comment:
You'll want to `source` this before running cucumber; or we could bring in dotenv for cucumber and rspec. Thoughts @bhaibel and @jtu0 ?

Reply:
I brought in .env to load .env.test in rspec and cucumber.